### DISCUSSION TITLE
Design

### DISCUSSION THREAD

**Discussion by sarjugarg** [jhuckaby/Cronicle#595] [Q&A]
hi,

I understand the cronicle run on one central server and then u can run worker process on multiple server which need to be monitored

When u schedule a job, it send HTTP command to worker thread to execute. Do we have internal architecture to understand

Can worker thread be removed and use something like sshpass to do the remote execution

Regards
Sarju

**Comment by jhuckaby** [comment]
Inner Workings → https://github.com/jhuckaby/Cronicle/blob/master/docs/InnerWorkings.md

It sounds like you want to use the Shell Plugin, and then SSH to your remote servers to run the commands you want.
